### ISSUE TITLE
Ontology text needs updating

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#750]
On https://openenergy-platform.org/ontology/, it still reads: 
_Here, the Open Energy Ontology OEO is being created as a domain ontology within the field of energy system modelling. The first release of the OEO is planned for summer 2020._

Since there is a regular release cylce now, we could update the text, so it does not need constant updating.
How about:
_The Open Energy Ontology OEO is created as a domain ontology within the field of energy system modelling. It is growing constantly and under a regular release cycle._ 

@stap-m @l-emele @sfluegel05 what do you think?

**Comment by l-emele** [comment]
As we have a [VERSION file](url) maybe it is possible to automatically include the version number of the latest release. Then we could add a sentence like: _The latest OEO version is x.y.z._

**Comment by han-f** [comment]
I like this idea @l-emele 
Extended suggestion: 
_The Open Energy Ontology OEO is created as a domain ontology within the field of energy system modelling. It is growing constantly and under a regular release cycle. The latest OEO version is x.y.z._

**Comment by stap-m** [comment]
Also a link to the ontology files could be included at that point.
The actual link is kind of "hidden" at the bottom of the page.

**Comment by han-f** [comment]
New suggestion:
_The Open Energy Ontology (OEO) is created as a domain ontology within the field of energy system modelling. It is growing constantly and under a regular release cycle. The latest OEO version is x.y.z. and its files can be assessed [here](https://openenergy-platform.org/ontology/oeo)._

**Comment by stap-m** [comment]
Nice! "accessed" instead of assessed?

**Comment by han-f** [comment]
> Nice! "accessed" instead of assessed?

Yes,  thanks freudscher Verschreiber - while in principle people can assess the files too  ;-)

Current suggestion:
_The Open Energy Ontology (OEO) is created as a domain ontology within the field of energy system modelling. It is growing constantly and under a regular release cycle. The latest OEO version is x.y.z. and its files can be accessed here._
